chart/kubeapps/README.md

```diff
@@ -286,3 +251,21 @@ kubectl apply -f <repo name>.yaml
 ```
 
 After that you should be able to access the new version of Kubeapps. If the above doesn't work for you or you run into any other issues please open an [issue](https://github.com/kubeapps/kubeapps/issues/new).
+
+### Upgrading to 2.0
+
+Kubeapps 2.0 (Chart version 4.0.0) introduces some breaking changes:
+
+ - Helm 2 is no longer supported. If you are still using some Helm 2 charts, [migrate them with the available tools](https://helm.sh/docs/topics/v2_v3_migration/). If you are facing any issue managing this migration and Kubeapps, please open a new issue!
```

Comment:
As per that link, "Helm 2 charts (apiVersion=v1) are still installable" and that page is more about migrating your environment rather than charts, though it does mention how to update a chart to the new v2 api. Again, I think it's good to point people here, but also be clear that many helm2 charts (v1) just continue to work when deployed with helm 3.